doc/source/upgrading_guide.rst

```diff
@@ -94,8 +94,20 @@ If you only used the |MyRocks| storage engine in |Percona Server| |version.prev|
 
    $ apt-get install percona-server-rocksdb
 
+.. deprecated:: 8.0.16-7
 
-The installation script will *NOT* run automatically :command:`mysql_upgrade` as
+The :command:`mysql_upgrade` was deprecated and removed as of |Percona Server|
+8.0.16-7. The functionality was moved to the `mysqld` binary which automatically
+upgrades, if needed.
```

Comment:
maybe better "The functionality was moved to the `mysqld` binary which automatically
upgrades, if needed." -> "The functionality was moved to the `mysqld` binary which automatically
runs upgrade process, if needed."
But I'll leave it to you since english is not my first language.